File: app/src/main/java/in/ceeq/lyte/databinding/ContactListItemViewModel.java
package in.ceeq.lyte.databinding;


import android.view.View;

import in.ceeq.lyte.ContactViewActivity;

public class ContactListItemViewModel extends Contact {

    public void onViewContactClick(View view) {
        ContactViewActivity.start(view.getContext(), getId());
    }
}

File: app/src/main/java/in/ceeq/lyte/databinding/ContactListItemViewModel.kt
package `in`.ceeq.lyte.databinding


import `in`.ceeq.lyte.user.UserProfileActivity
import android.view.View

class ContactListItemViewModel : Contact() {

    fun onViewContactClick(view: View) {
        UserProfileActivity.start(view.context, id)
    }
}
